Bottom: 16955da58186ad2a47b744c887a3e51c014ce063
Top:    df788e2c7b13720f8b8d56e19b4a619c5d723018
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 18:02:33 +0100

Refresh of hid-gtk-replace-usage-of-the-d

---

diff --git a/src/hid/gtk/gui-drc-window.c b/src/hid/gtk/gui-drc-window.c
index 4ab38b1..ce0210c 100644
--- a/src/hid/gtk/gui-drc-window.c
+++ b/src/hid/gtk/gui-drc-window.c
@@ -931,7 +931,7 @@ ghid_drc_window_show (gboolean raise)
 
   gtk_widget_realize (drc_window);
   if (Settings.AutoPlace)
-    gtk_widget_set_uposition (GTK_WIDGET (drc_window), 10, 10);
+    gtk_window_move (GTK_WINDOW (drc_window), 10, 10);
   gtk_widget_show_all (drc_window);
 }
 
diff --git a/src/hid/gtk/gui-library-window.c b/src/hid/gtk/gui-library-window.c
index a1e1a53..74b55e9 100644
--- a/src/hid/gtk/gui-library-window.c
+++ b/src/hid/gtk/gui-library-window.c
@@ -176,7 +176,7 @@ ghid_library_window_create (GHidPort * out)
 
   gtk_widget_realize (library_window);
   if (Settings.AutoPlace)
-    gtk_widget_set_uposition (GTK_WIDGET (library_window), 10, 10);
+    gtk_window_move (GTK_WINDOW (library_window), 10, 10);
 
   gtk_editable_select_region (GTK_EDITABLE
 			      (GHID_LIBRARY_WINDOW (library_window)->
diff --git a/src/hid/gtk/gui-log-window.c b/src/hid/gtk/gui-log-window.c
index 41571e0..01daaee 100644
--- a/src/hid/gtk/gui-log-window.c
+++ b/src/hid/gtk/gui-log-window.c
@@ -110,7 +110,7 @@ ghid_log_window_create ()
 
   gtk_widget_realize (log_window);
   if (Settings.AutoPlace)
-    gtk_widget_set_uposition (GTK_WIDGET (log_window), 10, 10);
+    gtk_window_move (GTK_WINDOW (log_window), 10, 10);
 }
 
 void
diff --git a/src/hid/gtk/gui-netlist-window.c b/src/hid/gtk/gui-netlist-window.c
index e05ae57..35b5427 100644
--- a/src/hid/gtk/gui-netlist-window.c
+++ b/src/hid/gtk/gui-netlist-window.c
@@ -838,7 +838,7 @@ ghid_netlist_window_create (GHidPort * out)
 
   gtk_widget_realize (netlist_window);
   if (Settings.AutoPlace)
-    gtk_widget_set_uposition (GTK_WIDGET (netlist_window), 10, 10);
+    gtk_window_move (GTK_WINDOW (netlist_window), 10, 10);
 
 }
 
diff --git a/src/hid/gtk/gui-pinout-window.c b/src/hid/gtk/gui-pinout-window.c
index 5b0feef..6433fb1 100644
--- a/src/hid/gtk/gui-pinout-window.c
+++ b/src/hid/gtk/gui-pinout-window.c
@@ -102,6 +102,6 @@ ghid_pinout_window_show (GHidPort * out, ElementType * element)
 
   gtk_widget_realize (top_window);
   if (Settings.AutoPlace)
-    gtk_widget_set_uposition (top_window, 10, 10);
+    gtk_window_move (GTK_WINDOW (top_window), 10, 10);
   gtk_widget_show_all (top_window);
 }
diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 06eab4d..2c76a0a 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -1685,7 +1685,7 @@ ghid_parse_arguments (int *argc, char ***argv)
 			       ghidgui->top_window_width, ghidgui->top_window_height);
 
   if (Settings.AutoPlace)
-    gtk_widget_set_uposition (GTK_WIDGET (window), 10, 10);
+    gtk_window_move (GTK_WINDOW (window), 10, 10);
 
   gtk_widget_show_all (gport->top_window);
   ghidgui->creating = TRUE;
